sources/packages/backend/libs/services/src/queue/model/queue.model.ts

```diff
@@ -3,4 +3,6 @@ import { BullModuleOptions } from "@nestjs/bull";
 export interface QueueModel extends Omit<BullModuleOptions, "redis"> {
   name: string;
   dashboardReadonly?: boolean;
+  isActive?: boolean;
```

Comment:
Is there a reason for these properties to be optional?
If not please make them not nullable.